embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -1518,8 +1526,13 @@
     }
 
     @Nullable
-    ActivityService getActivityService() {
-        return activityService;
+    ProcessStateService getActivityService() {
+        return processStateService;
+    }
+
+    @Nullable
+    ActivityTracker getActivityLifecycleTracker() {
```

Comment:
Who calls this? If we don't need this I'd rather not have it

Reply:
`EmbraceAutomaticVerification` calls it (and other services) unfortunately

Reply:
Ack....